services/crypto_delete.proto

```diff
@@ -29,18 +36,41 @@ option java_multiple_files = true;
 import "basic_types.proto";
 
 /**
- * Mark an account as deleted, moving all its current hbars to another account. It will remain in
- * the ledger, marked as deleted, until it expires. Transfers into it a deleted account fail. But a
- * deleted account can still have its expiration extended in the normal way.
+ * Delete an account.<br/>
+ * This will mark an account deleted, and transfer all tokens to a "sweep"
+ * account.
+ *
+ * A deleted account SHALL NOT hold a balance in any token type.<br/>
+ * A deleted account SHALL remain in state until it expires.<br/>
+ * Transfers that would increase the balance of a deleted account
+ * SHALL fail.<br/>
+ * A deleted account MAY be subject of a `cryptoUpdate` transaction to extend
+ * its expiration.<br/>
+ * When a deleted account expires it SHALL be removed entirely, and SHALL NOT
+ * be archived.
+ *
+ * ### Block Stream Effects
+ * None
  */
 message CryptoDeleteTransactionBody {
     /**
-     * The account ID which will receive all remaining hbars
+     * An account identifier.
+     * <p>
+     * The identified account SHALL receive all tokens from the deleted
```

Comment:
```suggestion
     * The identified account SHALL receive all tokens, token balances, and unique tokens from the deleted
```

Reply:
It only receives HBAR balance (at least that's my understanding) because an account cannot be deleted if it holds any other tokens.